src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -669,6 +670,38 @@ describe('Testing Edit concept actions ', () => {
     });
   });
 
+  it('should unretire a Mapping', () => {
```

Comment:
When should a mapping be unretired?

Reply:
I have changed it to `'should unretire a Mapping when the unretireMapping action is triggered'`